cpp/arcticdb/processing/operation_dispatch_binary.hpp

```diff
@@ -105,8 +105,10 @@ VariantData binary_membership(const ColumnWithStrings& column_with_strings, Valu
                     }
                 });
             } else {
-                util::raise_rte("Cannot check membership of {} in set of {} (possible categorical?)",
-                                column_with_strings.column_->type(), value_set.base_type());
+                util::raise_rte("Cannot check membership of {} {} in set of {} (possible categorical?)",
```

Comment:
Please replace all of these `util::raise_rte` with `E_INVALID_USER_ARGUMENT`

Reply:
Fixed in [a907db9](https://github.com/man-group/ArcticDB/pull/1557/commits/a907db94c4366371434d048c630f796fcbc7a60e)